Lesson19/Lesson19.cpp

```diff
@@ -0,0 +1,47 @@
+#include <iostream>
+#include "DynamicIntArray.h"
+
+int main()
+{
+    DynamicIntArray arr1;
+    DynamicIntArray arr2 = DynamicIntArray(10);
+
+    const int arr1Size = arr1.getSize();
+    for (int i = 0; i < arr1Size; i++) {
+        arr1.push_back(i * 10);
+        std::cout << "[" << arr1[i] << "] ";
+    }
+    std::cout << std::endl << std::endl;
+
+    const int arr2Size = arr2.getSize();
+    for (int i = 0; i < arr2Size; i++) {
+        arr2.push_back(i);
+        std::cout << "[" << arr2[i] << "] ";
+    }
+    std::cout << std::endl << std::endl;
+
+    const int arr2NewSize = 15;
+    arr2.setSize(arr2NewSize);
+    for (int i = 0; i < arr2NewSize; i++) {
+        arr2.push_back(i + 10);
+        std::cout << "[" << arr2[i] << "] ";
+    }
+    std::cout << std::endl << std::endl;
```

Comment:
check output, please. poes it work well?

Reply:
at the end it should print
0 1 2 3 4 5 6 7 8 9 10 11 12 13 14

Reply:
fixed, you right i misunderstood description of HW.